third-party/json.hpp

```diff
@@ -10374,7 +10374,8 @@ class basic_json
         */
         long double str_to_float_t(long double* /* type */, char** endptr) const
         {
-            return std::strtold(reinterpret_cast<typename string_t::const_pointer>(m_start), endptr);
+            using namespace std;
```

Comment:
Why is this required?

Reply:
There was a issue regarding this, look here: https://github.com/nlohmann/json/issues/136
I'll replace the json.hpp with the latest release.